NPB3.3-MPI/Dockerfile

```diff
@@ -0,0 +1,53 @@
+#The NBP Benchmark will run on ubuntu
+FROM ubuntu:latest
+
+
+#Get the NPB from http://www.nas.nasa.gov/publications/npb.html 
+RUN apt-get update && apt-get install -y\
```

Comment:
Don't forget to use the best practices when you apply such command.
https://docs.docker.com/engine/articles/dockerfile_best-practices/